project3/index.html

```diff
@@ -0,0 +1,387 @@
+            </svg>
+
+            <h1>SUPER HERO<br> REGISTRY</h1>
+            <h2 class="great-power">With great power there must also come-great responsibility</h2>
+        </header>
+
+
+
+        <div class="box box-2">
+
+            <h2 class="identity">IDENTITY</h2>
+            <form action="">
+
+                <div class="realname">
+                    <label class="input-box" for="real-name">Real Name</label>
+                    <input id="real-name" type="real-name" placeholder="" required>
+                </div>
+
+                <div class="alias">
+                    <label class="input-box" for="alias">Alias</label>
+                    <input id="alias" type="alias" placeholder="" required>
+                </div>
+
+                <div class="email">
+                    <label class="input-box" for="email" Email>Email</label>
+                    <input class="email" type="email" type="email" placeholder="" required>
+
+                </div>
+
+                <h2 class="personal">PERSONAL CHARACTERISTICS</h2>
+
+
+                <label class="age" for="age">Age<br></label>
+                <select class="agebox" name="" id="age">
+                    <option value="testing"></option>
+                    <option value="another"></option>
+                </select>
+
+                <fieldset class="field-margin">
+                    <legend class="category">Category</legend>
+
+                    <p class="margins">Please select as many as apply to your origin and how you gained your powers.</p>
+
+                    <input name="category" id="alien" type="checkbox">
+                    <label for="alien">Alien</label><br>
+
+                    <input name="category" id="cyborg" type="checkbox">
+                    <label for="cyborg">Cyborg</label><br>
+
+                    <input name="category" id="experiment" type="checkbox">
+                    <label for="experiment">Experiment</label><br>
+
+                    <input name="category" id="human" type="checkbox">
+                    <label for="human">Human</label><br>
+
+                    <input name="category" id="inhuman" type="checkbox">
+                    <label for="inhuman">Inhuman</label><br>
+
+                    <input name="category" id="mutant" type="checkbox">
+                    <label for="mutant">Mutant</label><br><br>
+                </fieldset>
+
+                <fieldset class="field-margin">
+                    <legend>Gender</legend>
+                    <input name="Gender" id="woman" type="radio">
+                    <label for="woman">Woman</label><br>
+                    <input name="Gender" id="man" type="radio">
+                    <label for="man">Man</label><br>
+                    <input name="Gender" id="nonbinary" type="radio">
+                    <label for="nonbinary">Nonbinary</label><br>
+                    <input name="Gender" id="genderqueer/genderfluid" type="radio">
```

Comment:
An `id` cannot have a space or slash (`/`) in it -- only single words or words combined via dashes (`-`) or underscores (`_`). Then, whatever fix you make to this `id`, you'd also have to update the `for` attribute of the `label` on line 99.